src/index.js

```diff
@@ -5,23 +5,33 @@
 import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png';
+// import './images/turing-logo.png';
 import moment from 'moment';
 import fetchHandler from './fetchHandler.js';
 import Traveler from './Traveler.js';
 import Trip from './Trip.js';
 import Destination from './Destination.js';
 import domUpdates from './domUpdates.js';
 // console.log('This is the JavaScript entry file - your code begins here.');
-window.addEventListener('load', getData())
+
+
+window.addEventListener('load', getData());
 
 let user;
 let allTravelers;
 let allTrips;
 let allDestinations;
+let newTripData;
+
+const submitButton = document.querySelector('.submit-button');
+submitButton.addEventListener('click', function() {
+  getNewTripData(user, allDestinations);
+  fetchHandler.fetchPostTrip(newTripData);
+  getData();
+});
```

Comment:
Just as an observation, in mod 1 they asked us to not use this syntax when creating eventListeners, but I am sure that if you have used it before during this mod I don't think it will be a problem